bundles/org.openhab.binding.melcloud/NOTICE

```diff
@@ -0,0 +1,15 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
+
+https://github.com/openhab/openhab2-addons
+
+== Third-party Content
```

Comment:
This line `Third-party Content` and the empty line above can also be removed from the NOTICE file.